apps/torus-governance/src/app/components/proposal/create-proposal.tsx

```
@@ -3,6 +3,7 @@
 import { zodResolver } from "@hookform/resolvers/zod";
 import type { TransactionResult } from "@torus-ts/torus-provider/types";
 import { toast } from "@torus-ts/toast-provider";
+import type { TransactionResult } from "@torus-ts/torus-provider/types";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TransactionResult'. Consider to delete it or rename it.

'TransactionResult' is defined here:


(lint/suspicious/noRedeclare)